rotkehlchen/data_migrations/migrations/migration_5.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def data_migration_5(write_cursor: 'DBCursor', rotki: 'Rotkehlchen') -> None:  # noqa: E501 # pylint: disable=unused-argument
+    """
+    Update view for amm_swaps making sure that the link is a string instead of blob
+    """
+    log.debug('Starting migration 5')
+    write_cursor.execute('DROP VIEW combined_trades_view;')
```

Comment:
This data migration is a problematic. It's not a migration but a schema upgrade. This breaks many of our expectations on how rotki works during migrations and schema upgrades.

Have you considered what will happen when we remove the combined trades view? This migration will need to be skipped.

With something like 

```sql
SELECT name FROM sqlite_master WHERE type='view' AND name='combined_trades_view';
```

and checking the result exists (do migration) or not (don't do it).

But still though possible and would probably work this breaks our assumption that migrations do not change schema, just move data around.

If something goes wrong for example no backup is going to exist to bail the user out.

Also as I mentioned in the tests section if you go with this you would need to add a test for the case of view missing.